lib/Rex/Commands/Sysctl.pm

```diff
@@ -95,10 +99,14 @@ sub sysctl {
       Rex::Logger::debug("$key has already value $val");
     }
 
-    if ( $options{persistent} ) {
+    if ( $options{ensure} eq "present" ) {
```

Comment:
Yes it was throwing an error if not using ensure. I committed a change to check if ensure is used, and also I added a check to be give an error if it's not absent/present, so if you type "presetn" you will know something is wrong.